Framework/PythonInterface/plugins/algorithms/LoadElementalAnalysisData.py

```diff
@@ -0,0 +1,179 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import glob
+import os
+import numpy as np
+from mantid import config
+from mantid.api import AlgorithmFactory, AnalysisDataService, PythonAlgorithm, TextAxis, WorkspaceGroup, WorkspaceGroupProperty
+from mantid.kernel import Direction, IntBoundedValidator
+from mantid.simpleapi import ConvertToHistogram, ConvertToPointData, DeleteWorkspace, LoadAscii, WorkspaceFactory
+
+
+type_keys = {"10": "Prompt", "20": "Delayed", "99": "Total"}
```

Comment:
Can we uppercase these variable names to follow the standards? It makes it more obvious in the code when they are used that they are module-scope variables

Reply:
Resolved in commit fd81a40

Reply:
Resolved in commit fd81a40